<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wepConfig</class>
 <widget class="QDialog" name="wepConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>374</width>
    <height>285</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WEP Configuration</string>
  </property>
  <property name="windowIcon">
   <iconset resource="wificonfig.qrc">
    <normaloff>:/tray_wifi85.png</normaloff>:/tray_wifi85.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Wireless Network Key</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="3">
       <widget class="QRadioButton" name="radioHex">
        <property name="text">
         <string>Hex Key</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="3" colspan="2">
       <widget class="QRadioButton" name="radioPlaintext">
        <property name="text">
         <string>Plaintext</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="textLabel1">
        <property name="text">
         <string>Network Key</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="2" colspan="3">
       <widget class="QLineEdit" name="lineKey">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="textLabel1_2_2">
        <property name="text">
         <string>Key Index</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="3">
       <widget class="QSpinBox" name="spinIndex">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>4</number>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <spacer name="spacer11">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>61</width>
          <height>21</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="0" colspan="5">
       <widget class="QCheckBox" name="checkShowKey">
        <property name="text">
         <string>Show Key</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <includes>
  <include location="local">qmessagebox.h</include>
  <include location="local">QGroupBox</include>
 </includes>
 <resources>
  <include location="wificonfig.qrc"/>
 </resources>
 <connections/>
</ui>
